pom.xml

```diff
@@ -76,7 +76,12 @@
         <xmlsec.version>2.0.8</xmlsec.version>
         <jts.version>1.12</jts.version>
         <jts.bundle.version>1.12_1</jts.bundle.version>
+
+        <!-- If you -->
```

Comment:
:question: Do we need to leave a comment here stating that if you upgrade Karaf, you may need to update other versions in other files (startup.properties, felix config admin version, etc.)?